### PR TITLE
add: dataset generation code; add: random noise perturbation baseline

### PR DESCRIPTION
**Pull request by zlxlty**
add: dataset generation code
add: random noise perturbation baseline

### CODE REVIEW

File: attacks/locsearchadv/method.py [hunk 1]
```diff
@@ -15,51 +15,6 @@ class LocSearchAdv(AttackMethod):
     LB = -1 #This is just a guess for what the lower and upper bounds should be
     UB = 1  #This is just a guess for what the lower and upper bounds should be
 
-    def cyclic(self, I, r, b, x, y):
```
Comment (tutingjun) [inline attacks/locsearchadv/method.py:18]: Next time please don't change the code directly. Ask and let me and Alice do the changes as a way to show respect to others work!

**Comment by tutingjun** [comment]
Do we need eval and image folder for the repo?

File: attacks/locsearchadv/method.py (hunk 1 above)
Comment (zlxlty) [reply attacks/locsearchadv/method.py:18]: I switched the position of do_perturbation function to the front to take a screen shot. Sorry for not changing it back. The content of any functions you wrote remain intact.

**Comment by zlxlty** [comment]
> Do we need eval and image folder for the repo?

Ideally, we should have a place to openly store all our original experiment data. The images are too big for GitHub but I think it's fine to store eval json files here.

**Comment by tutingjun** [comment]
> > Do we need eval and image folder for the repo?
> 
> Ideally, we should have a place to openly store all our original experiment data. The images are too big for GitHub but I think it's fine to store eval json files here.

That's a good point. Can you then remove the image folder and keep the eval folder?

**Comment by zlxlty** [comment]
> > > Do we need eval and image folder for the repo?
> > 
> > 
> > Ideally, we should have a place to openly store all our original experiment data. The images are too big for GitHub but I think it's fine to store eval json files here.
> 
> That's a good point. Can you then remove the image folder and keep the eval folder?

No problem, I'm still add comments to my part of the code and will remove the image folder with the next pull request

**Review by tutingjun — APPROVED** [review]

**Pull request merged**